lib/requests_signature_dart.dart

```diff
@@ -1,3 +1,3 @@
 library requests_signature_dart;
 
-export 'src/dhmac_requests.dart';
+//export 'src/requests_signature_interceptor.dart';
```

Comment:
Shouldn't there be some content here?

Reply:
there should, that's after we finish migrating HTTP.CLIENT package

Reply:
maybe @Soap-141 or @Lee31416 could point me in the right direction?

Reply:
I'm not sure how I can help you since I have little context but what jp said makes sense to me

Reply:
Well looking at other packages as an example, it seems like you could simply export all files?

Example: https://github.com/cfug/dio/blob/main/dio/lib/dio.dart